src/OmiseCard.js

```diff
@@ -28,13 +28,15 @@ export const defaultIframeAppConfig = {
   submitLabel: 'Pay',
   buttonLabel: 'Pay with Omise',
   location: 'no',
+  address: 'auto', // auto, extended, simple
```

Comment:
I think we should explicit this variable name like @danfowler suggested.

```suggestion
  billingAddress: 'auto', // auto, extended, simple
```

Reply:
Yeah, I think so